metaflow/cli.py

```diff
@@ -354,19 +382,76 @@ def start(
     # When we process the options, the first one processed will return None and the
     # second one processed will return the actual options. The order of processing
     # depends on what (and in what order) the user specifies on the command line.
-    config_options = config_file_options or config_value_options
-    ctx.obj.flow = ctx.obj.flow._process_config_decorators(config_options)
+    config_options = config_file or config_value
+
+    if (
+        hasattr(ctx, "saved_args")
+        and ctx.saved_args
+        and ctx.saved_args[0] == "resume"
+        and getattr(ctx.obj, "has_config_options", False)
+    ):
+        # In the case of resume, we actually need to load the configurations
+        # from the resumed run to process them. This can be slightly onerous so check
+        # if we need to in the first place
+        if getattr(ctx.obj, "has_cl_config_options", False):
```

Comment:
what's cl?

Reply:
command line. I can make it longer. Basically, this part of the code is only hit if the user explicitly specified a `--config` or `--config-value` option (as opposed to being derived from defaults).